sfs/util.py

```diff
@@ -491,3 +496,26 @@ def _count_walls_1d(a):
         wall_count = wall_count[N_mask, :]
 
     return xs, wall_count
+
+
+def spherical_hn2(n, z):
+    r"""Spherical Hankel function of 2nd kind.
+
+    Defined as http://dlmf.nist.gov/10.47.E6,
+
+    .. math:: \hankel{2}{n}{z} = \sqrt{\frac{\pi}{2z}} \Hankel{2}{n + 1/2}{z},
+
+    where :math:`\Hankel{2}{n}{}` is the Hankel function of the second kind.
```

Comment:
I'd prefer a `\cdot` in the parentheses, but how is this handled in the SFS theory?

Reply:
In most cases in SFS theory I wrote it without any argument, e.g. `f(x) = g(x) + h(x), where `g` is a function. For the Hankel function this is not possible as we have to provide a third argument. In this case I agree with you that adding a `\cdot` should be better.